client/src/components/Profile/__test__/MainCard.test.tsx

```diff
@@ -1,44 +1,28 @@
 import React from 'react';
 import { render } from '@testing-library/react';
+import { ProfileMainCardData } from 'services/user';
 import MainCard from '../MainCard';
-import { ProfileInfo } from 'services/user';
 
 // TODO: Known Issue: https://stackoverflow.com/questions/59942808/how-can-i-use-jest-coverage-in-next-js-styled-jsx
 
-const mockData = {
-  generalInfo: {
-    name: 'John Doe',
-    githubId: 'john-doe',
-    location: {
-      countryName: 'Belarus',
-      cityName: 'Minsk',
-    },
+const mockData: ProfileMainCardData = {
+  name: 'John Doe',
+  githubId: 'john-doe',
+  location: {
+    countryName: 'Belarus',
+    cityName: 'Minsk',
   },
   publicCvUrl: 'public-url',
-} as ProfileInfo;
+};
 
 describe('MainCard', () => {
   describe('Should render correctly', () => {
     it('if is editing mode disabled', () => {
```

Comment:
Not your changes, but would be nice to replace with "if editing mode is disabled"